lib/rabbitmqctl.ex

```diff
@@ -229,4 +233,18 @@ defmodule RabbitMQCtl do
     :net_kernel.stop
     exit({:shutdown, code})
   end
+
+  @spec with_distribution(options(), (() -> command_result())) :: command_result()
+  defp with_distribution(options, code) do
+    # Tries to start net_kernel distribution, and calls `code`
+    # function on success. Otherswise returns error suitable for
+    # handle_shutdown/0.
+    case Distribution.start(options) do
+      :ok ->
```

Comment:
This is unpacked at 
https://github.com/binarin/rabbitmq-cli/blob/71d753b8088317d91763c0103fcec4d5b7027ce2/lib/rabbitmq/cli/core/distribution.ex#L49